src/arc_utilities/xbox.py

```diff
@@ -1,9 +1,18 @@
+import numpy as np
```

Comment:
We should include a note about which driver this uses. We've gotten confused in the past when all the buttons/axes switch indices.